test/integration-tests/runner/javascript/test_runner.js

```diff
@@ -1,25 +1,29 @@
+// TODO: We need to fix this Jochen
 const iotea = require('../../../../src/module.js');
-//const iotea = require('boschio.iotea');
+// const iotea = require('boschio.iotea');
 
-process.env.MQTT_TOPIC_NS = 'iotea/';
-
-const Logger = iotea.util.Logger;
-process.env.LOG_LEVEL = Logger.ENV_LOG_LEVEL.INFO;
+const {
+    TestRunnerTalent,
+    ProtocolGateway
+} = iotea;
 
 const {
-    TestRunnerTalent
-} = require('../../../../src/core/talent.test.js')
+    Logger
+    MqttProtocolAdapter
+} = iotea.util;
+
+process.env.LOG_LEVEL = Logger.ENV_LOG_LEVEL.INFO;
 
 class TestRunner extends TestRunnerTalent {
-    constructor(connectionString) {
+    constructor(protocolGatewayConfig) {
         // Define your testSetTalent list and set via super constructor
-        super('testRunner-js', ['testSet-sdk-js', 'testSet-sdk-py', 'testSet-sdk-cpp'], connectionString);
-        
-        // you can run singular tests say for development also 
-        //super('testRunner-js', ['testSet-sdk-js'], connectionString);
+        super('testRunner-js', ['testSet-sdk-js', 'testSet-sdk-py', 'testSet-sdk-cpp'], protocolGatewayConfig);
+
+        // you can run singular tests say for development also
+        //super('testRunner-js', ['testSet-sdk-js'], protocolGatewayConfig);
     }
 }
 
-const runner = new TestRunner('mqtt://localhost:1883');
+const runner = new TestRunner(ProtocolGateway.createDefaultConfiguration([ MqttProtocolAdapter.createDefaultConfiguration() ]));
```

Comment:
The 'hardcoded' default configuration will be switched to a 'test' specific configuration. But for now, this is ok.

Reply:
I did not want to change anything of the current behavior. We can thing about it, sure.